ui/src/app/archived-workflows/components/archived-workflow-list/archived-workflow-list.tsx

```diff
@@ -181,8 +181,54 @@ export class ArchivedWorkflowList extends BasePage<RouteComponentProps<any>, Sta
         return params;
     }
 
+    private filterStateObject(state: State, whitelistedStates: string[], blacklistedStates: string[]) {
+        const filteredState: State = {} as State;
+
+        if (whitelistedStates.includes('pagination') || !blacklistedStates.includes('pagination')) {
+            filteredState.pagination = state.pagination;
+        }
+
+        if (whitelistedStates.includes('namespace') || !blacklistedStates.includes('namespace')) {
+            filteredState.namespace = state.namespace;
+        }
+
+        if (whitelistedStates.includes('name') || !blacklistedStates.includes('name')) {
+            filteredState.name = state.name;
+        }
+
+        if (whitelistedStates.includes('namePrefix') || !blacklistedStates.includes('namePrefix')) {
+            filteredState.namePrefix = state.namePrefix;
+        }
+
+        if (whitelistedStates.includes('selectedPhases') || !blacklistedStates.includes('selectedPhases')) {
+            filteredState.selectedPhases = state.selectedPhases;
+        }
+
+        if (whitelistedStates.includes('selectedLabels') || !blacklistedStates.includes('selectedLabels')) {
+            filteredState.selectedLabels = state.selectedLabels;
+        }
+
+        if (whitelistedStates.includes('minStartedAt') || !blacklistedStates.includes('minStartedAt')) {
+            filteredState.minStartedAt = state.minStartedAt;
+        }
+
+        if (whitelistedStates.includes('maxStartedAt') || !blacklistedStates.includes('maxStartedAt')) {
+            filteredState.maxStartedAt = state.maxStartedAt;
+        }
+
+        if (whitelistedStates.includes('error') || !blacklistedStates.includes('error')) {
+            filteredState.error = state.error;
+        }
+
+        if (whitelistedStates.includes('deep') || !blacklistedStates.includes('deep')) {
+            filteredState.deep = state.deep;
+        }
+
```

Comment:
I'm a little confused. Shouldn't `workflows` be in here? Otherwise, passing it in here seems to have no effect